03b_model/src/run_model.py

```diff
@@ -724,21 +729,21 @@ def test_hyperparameters():
             print('Running hyperparameter-replicate combination '+str((j+1)*(i+1))+' of '+str(len(hp_tune_vals)*hp_config['replicates']))
 
             run_id = os.path.join(hp_id, str(i).rjust(2,'0'))
-            
+            seed = i
             train_model(prepped_model_io_data_file, inputs, seq_len,
```

Comment:
You have some commented out code that uses the train_model() and make_predictions() functions. Should those be updated to get the new inputs too, if there a possibility that you would uncomment them at some point?

Reply:
Removed the commented code, it was old and we won't be using it.